proto/gogoproto/gogo.pb.go

```diff
@@ -0,0 +1,888 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: gogoproto/gogo.proto
+
+package gogoproto
```

Comment:
@bufdev when I use buf generate it adds this file. Could you touch on why this happens?

Reply:
ah looks like this is it:

```
// This is a compile-time assertion to ensure that this generated file
// is compatible with the proto package it is being compiled against.
```

Reply:
It's because `buf generate` is touching all your roots, we're working on adding a `--only` (name TBD) flag that lets you filter - stand by

Reply:
ah okay. I was looking for info on that in the docs but assumed is would ignore third_party. Will revert to using protoc for the time being.

Reply:
We may have this feature out by EOD, or end of tomorrow at least, just FYI, if you want to hold off :-)

Reply:
Wow, that's fast! Will leave this PR here for till tomorrow in anticipation. 😄

Reply:
Sorry for delay - this is complete internally, we'll have a release with this on Monday!

Reply:
Released as v0.31.0 - you can now filter by directory via `buf generate --path path/to/dir`, the documentation has been updated as well!

Reply:
Amazing, thank you. Rebuilding our docker image now to use 0.31